common/src/main/java/com/wynntils/features/chat/ChatTabsFeature.java

```diff
@@ -196,6 +196,8 @@ public void onDisable() {
 
     @Override
     protected void onConfigUpdate(Config<?> config) {
+        if (!isEnabled()) return;
```

Comment:
I assume onConfigUpdate is called AFTER isEnabled is already set (so after the config is already set).